src/content/docs/tutorials/adding_a_pipeline/first_release.md

```diff
@@ -10,30 +10,38 @@ markdownPlugin: 'checklist'
 
 ## Reset the default branch
 
-When the code is stable and ready for a release, set the `master` branch to be the default
-branch again.
+When the code is stable and ready for a release, set the default branch to `master`.
 
 ## Bump the version
 
-At this point you should bump the version numbers on `dev`.
+Use `nf-core bump-version` to increment the version number on the `dev` branch to remove
+the `dev` suffix.
 
-When developing the pipeline, the version numbers should be numeric with `dev` at the end.
-Use the `nf-core bump-version` command to do this - there are quite a few locations in the
-code that need updating and this ensures that it happens in the correct places.
+When developing the pipeline, the version numbers should be numeric with `dev` at the end,
+e.g. `0.0.0dev`. When making a release, version numbers should all be numeric. Pipeline
+release numbers MUST use [Semantic Versioning](https://semver.org/).
 
-When making a release, version numbers should all be numeric. Use `nf-core lint --release`
-when ready - this will check that everything looks correct. Pipeline release numbers MUST
-use [Semantic Versioning](https://semver.org/).
+:::note
+Use the `nf-core bump-version` command to increment the version numbers. The version number
+exists in many places in the codebase, and this tool consistently updates all of them.
+:::
```

Comment:
```suggestion
:::warning
Instead of manually changing the version number, use the `nf-core bump-version` command to increment the version numbers. The version number
exists in many places in the codebase, and this tool consistently updates all of them.
:::
```

Reply:
Feel like this is more of a tip than a warning? I mean, you _can_ update them all manually if you want. Linting should fail if you mess it up I think..

```suggestion
:::tip
Instead of manually changing the version number, use the `nf-core bump-version` command to increment the version numbers. The version number
exists in many places in the codebase, and this tool consistently updates all of them.
:::
```